src/software/backend/backend.h

```diff
@@ -17,10 +17,21 @@
 class Backend : public Subject<SensorMsg>,
                 public Subject<World>,
                 public Subject<RobotStatus>,
+                public ThreadedObserver<World>,
                 public ThreadedObserver<ConstPrimitiveVectorPtr>
 {
    public:
     Backend() = default;
 
     virtual ~Backend() = default;
+
+    /**
+     * Callback function to send components of SensorMsg via Subject<SensorMsg>
```

Comment:
* Callback functions to send components of SensorMsg via Subject<SensorMsg>

Reply:
Would be helpful to clarify the exact side effects of these functions here. Do we always send a sensor msg? Only after we've receive one of each message?

Reply:
clarified